packages/ui/src/stories/formkit/Datepicker.stories.ts

```diff
@@ -0,0 +1,24 @@
+import { StoryFn } from '@storybook/vue3'
```

Comment:
Here we probably need Date and Time components (Date Input and Time Input from vuestic).

I see no Reason for other components.

Also Date and DatetimeLocal components look wrong. We're providing FormKit wrapper for Vuestic components, so all components should look as Vuestic components.

<img width="374" alt="image" src="https://github.com/user-attachments/assets/e8c905d0-1002-45f0-9bb4-0e98fbcea436" />